internal/login/handler/register_handler.go

```diff
@@ -0,0 +1,115 @@
+package handler
+
+import (
+	"github.com/caos/zitadel/internal/auth_request/model"
+	caos_errs "github.com/caos/zitadel/internal/errors"
+	usr_model "github.com/caos/zitadel/internal/user/model"
+	"golang.org/x/text/language"
+	"net/http"
+)
+
+const (
+	tmplRegister = "register"
+
+	globalRO = "GlobalResourceOwner"
```

Comment:
does this work? maybe we need to configure resourceowner on template (on zitadel template it should be global)